### PR TITLE
Nep 292 user preference

### PR DESCRIPTION
**Pull request by ShivaAryal**
-   [x] Include Jira Ticket No. in Title
-   [ ] Write Unit Tests
-   [ ] Add screenshots whenever possible
-   [x] Make sure Github Action build succeed
-   [x] Address Automated Code Reviews from LGTM, Deepscan and Synk
-   [x] Add Reviewers

### CODE REVIEW

File: src/services/readArticles.service.js [hunk 1]
```diff
@@ -3,14 +3,15 @@ import gql from 'graphql-tag'
 import crashlytics from '@react-native-firebase/crashlytics'
 
 class ReadArticleService{
```
Comment (syuraj) [inline src/services/readArticles.service.js:5]: @ShivaAryal Please, write some tests around these services.

File: src/services/asyncStorageService.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ const saveReadArticles = (article) => {
             articles.push(article)
```
Comment (syuraj) [inline src/services/asyncStorageService.js:10]: @ShivaAryal Please, write some tests around these services.

**Review by syuraj — CHANGES_REQUESTED** [review]

File: src/services/readArticles.service.js (hunk 1 above)
Comment (ShivaAryal) [reply src/services/readArticles.service.js:5]: app mai code lekhne vannuvako?

Comment (ShivaAryal) [reply src/services/readArticles.service.js:5]: @syuraj

**Pull request merged**